functional/fixtures/units/single@.service

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStart=/bin/bash -c "while true; do echo Hello, World!; sleep 1; done"
```

Comment:
Hm... The test doesn't use the output at all; but I guess it might potentially be helpful for debugging.